ramls/item.json

```diff
@@ -69,6 +69,12 @@
           "type": "string",
           "description": "Effective Call Number Suffix is the suffix of the identifier assigned to an item or its holding and associated with the item.",
           "readonly": true
+        },
+        "typeId": {
+          "type": "string",
+          "description": "Effective Call Number Type Id is the call number type id assigned to the item or associated holding.",
```

Comment:
Can the description be clear which id is chosen if both item and holding have such an id?
For example "is the call number type id of the item, if available, otherwise that of the holding."